src/Integration/RequestIntegration.php

```diff
@@ -169,13 +169,16 @@ private function processEvent(Event $event, Options $options): void
     /**
      * Removes headers containing potential PII.
      *
-     * @param array<string, string[]> $headers Array containing request headers
+     * @param array<mixed, string[]> $headers Array containing request headers
```

Comment:
You could either do
```suggestion
     * @param array<array-key, string[]> $headers Array containing request headers
```
or 
```suggestion
     * @param string[] $headers Array containing request headers
```

Reply:
I'm not sure what you mean by "surface at the static analysis level", but I do like using `array-key` allows me to drop the added entry to the baseline file.

Reply:
I meant exactly that :smile: if it was in the baseline, the static analysis was already picking this bug, it was just silenced.